libdeno/binding.cc

```diff
@@ -530,7 +530,7 @@ void DenoIsolate::ResolveOk(const char* filename, const char* source) {
 }
 
 bool ExecuteMod(v8::Local<v8::Context> context, const char* js_filename,
-                const char* js_source) {
+                const char* js_source, int is_prefetch) {
```

Comment:
Maybe the name of this argument should be changed at this level level of abstraction, because in libdeno we don't know about "prefetching" ... 

What do you think about "resolve_only" or "without_execute"... something like that?

Also you may use a `bool` here as this is C++.

Reply:
Adopting `resolve_only` (`execute_mod` that `without_execute` sounds weird...)